app/components/common/SystemCell.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import {
+  View,
+  StyleSheet,
+  TouchableOpacity,
+  ImageBackground,
+  Dimensions,
+} from 'react-native'
+import RNText from './RNText'
+import colors from '../../config/colors'
+
+const windowWidth = Dimensions.get('window').width
+
+const SystemCell = ({ imageUrl, icon, title, onPress }) => {
+  const image = { uri: imageUrl }
+
+  return (
+    <TouchableOpacity
+      onPress={onPress}
+      accessible={true}
+      style={styles.container}
+    >
+      <ImageBackground
+        imageStyle={{ borderRadius: 6 }}
+        source={image}
+        style={styles.image}
+      >
```

Comment:
I think we should try use FastImage? Can merge with this but I'd add a todo to look to swap it out later. Also add a fallback image or even a backgroundColor.